HIP/hip-551.md

```diff
@@ -2,126 +2,224 @@
+ 3. Have their own payer.
+ 4. Are deduplicated on an individual basis. If two inner transactions within the same batch have the same `TransactionID`, the transaction will be rejected.
+ 5. `TransactionID` of an inner transaction has `scheduled`=`false` and `nonce`=`0`.
+ 6. Are authorized solely by their keys; batch keys are not used for authorization.
+ 7. Cannot be batch transactions. A batch transactions cannot contain another batch transaction.
+ 8. Cannot be network freeze transactions, but may be other privileged transactions.
+ 9. Must have nodeAccountID set to 0.0.0. This acts as a marker to indicate that an inner transaction is part of a batch, and it allows resubmitting a batch transaction without the need to re-sign the inner transactions. 
+10. Must set batchKey.
 
-The batch:
+The AtomicBatch outer transaction:
 
 1. Has its own payer and signatures.
 2. Considers inner transactions as part of its signed bytes.
 3. Makes the batch payer cover node+network fees for handling the batch, excluding inner transaction fees.
 4. Has deduplication based on its unique transaction ID.
-5. The submitter does not need to be authorized to carry out all the inner batch transactions. As with every transaction in Hedera, where the submitter can be any account as long as the transaction is properly signed.
-6. The standard max transaction size must be respected.
+5. The submitter does not need to be authorized to carry out all the inner batch transactions as with every transaction in Hedera. The submitter can be any account as long as the transaction is properly signed.
+6. Must not set batchKey.
+7. Must not be scheduled to run at a future time via ScheduleCreate.
+8. Must contain at least one inner transaction.
 
-We could create a new protobuf message based on `SignedTransaction` message:
-```
-message BatchTransaction {
-    /**
-    * TransactionBodies serialized into bytes, which must be signed
+### Batch Transaction Processing and Output
+
+- The batch transaction will be processed as a single unit. If any inner transaction fails, the entire batch will fail.
+- The batch transaction will return a single transaction response as usual. In general, inner transactions are not checked during pre-check.
+- The batch transaction and all executed inner transactions create TransactionReceipts. In order to see response codes for inner transactions, the user must query the individual inner transactions receipts.
+- Transaction records for inner transactions will have the `parentConsensusTimestamp` set to the `consensusTimestamp` of the containing AtomicBatch transaction. The `parentConsensusTimestamp` can be used to correlate all transaction records for the batch. Block streams are handled similarly.
+
+The following diagram illustrates the relationship between the transactions and the produces record stream:
+
+![Diagram of six transactions and the resulting record stream](../assets/hip-551/record-stream.png)
```

Comment:
The diagram should have children Tc1 and Tc2 having a parentConsensusTimestamp of Ti2, not n the grand parent. Otherwise the actual parent relationship hierarchy is lost.